content/logs/explorer/patterns.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Log Patterns (Beta)
+kind: documentation
+description: "Spot Log Patterns"
+aliases:
```

Comment:
@l0k0ms why doesn't this have beta metadata set?

Reply:
https://github.com/DataDog/documentation/pull/3162